Composer/packages/client/src/components/AddRemoteSkillModal/CreateSkillModal.tsx

```diff
@@ -196,16 +198,22 @@ export const CreateSkillModal: React.FC<CreateSkillModalProps> = (props) => {
     TelemetryClient.track('AddNewSkillCompleted');
     // if added remote skill fail, just not addTrigger to root.
     const skillId = location.href.match(/skill\/([^/]*)/)?.[1];
+
+    //if the root dialog is orchestrator recoginzer type or user chooses orchestrator type before connecting,
```

Comment:
Adding to the root dialog should not depending on the recognizer type. No matter it is luis, orchestrator, regex, we should add the trigger to the root dialog, right?